### PR TITLE
added import functions

### PR DESCRIPTION
**Pull request by OraCool**

### CODE REVIEW

File: serverless.ts [hunk 1]
```diff
@@ -27,11 +27,11 @@ const serverlessConfiguration: Serverless = {
     },
     environment: {
       AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
-      PG_HOST: '',
+      PG_HOST: 'rs-aws-db.coxauduykchc.eu-west-1.rds.amazonaws.com',
       PG_PORT: 5432,
       PG_DATABASE: 'rsnodedb',
       PG_USERNAME: 'postgres',
-      PG_PASSWORD: '',
+      PG_PASSWORD: 'ssg23121995',
```
Comment (alexeyMohnatkin) [inline serverless.ts]: you shouldn't keep password in the repository

**Comment by alexeyMohnatkin** [comment]
no links are provided, impossible to check

File: apps/api/src/import-service/handlers/importFileParser.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  console.log(
+    `importFileParser. File to parse: ${JSON.stringify({ path: key })}`
+  );
+  const results = [];
+  const stream = s3.getObject(params).createReadStream();
+  console.log(
+    `importFileParser. After get object: ${JSON.stringify({ obj: stream })}`
+  );
+  const status = stream
+    // .pipe(csv())
```
Comment (spiderpoul) [inline apps/api/src/import-service/handlers/importFileParser.ts:44]: why it's commented?

File: apps/api/src/import-service/handlers/importProductsFile.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    headers: {
+      'Access-Control-Allow-Origin': '*',
+    },
+    statusCode: 200,
+    body: 'imported',
+  };
+  if (
+    event &&
+    event.queryStringParameters &&
+    event.queryStringParameters.name
```
Comment (spiderpoul) [inline apps/api/src/import-service/handlers/importProductsFile.ts]: you could use optional chaining in ts `event?.queryStringParameters?.name`

**Pull request merged**